### DISCUSSION TITLE
Unable to render figure shortcode within quote shortcode

### DISCUSSION THREAD

**Discussion by dsiguero** [CaiJimmy/hugo-theme-stack#852] [Q&A] [answered]
```hugo
# post/foobar/index.md
---
<frontmatter stuff...>
---

{{< quote author="A famous person" source="The book they wrote" url="https://en.wikipedia.org/wiki/Book">}}
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
{{< figure src="image.jpg" title="An elephant at sunset" >}}
{{< /quote >}}
```

`image.jpg` lives in the same folder (`post/foobar/`) than `index.md`. If set in frontmatter's `image`, it loads fine. Also loads fine if the exact same shortcode is placed outside the `quote` shortcode.

**Comment by Lovirr** [comment]
i write down your codes above in my blog but it seems works properly? 

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/1eaccb0d-1229-481e-9be8-049acdb4f5c4)

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/fd2d8629-1168-4c7c-9822-9bdae7648d02)

💡have you  filled the `baseurl` in the `config.yaml`？

**Comment by dsiguero** [comment]
Yes, `baseurl` is correct, although I should have clarified that I'm seeing this locally, when working with draft pages.

If I set the image outside the quote shortcode, it renders correctly: `<img src="image.jpg>`.

However, when I set the image inside the quote shortcode, doesn't render anything, just `<!-- raw HTML omitted -->`.

I will try to reproduce it setting up a repo with the starter template.

**Reply by Lovirr** [reply, marked as answer]
i might realize why this happend in you blog
just checkout the `config.yaml`, have you set this to true?
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/70d5ff46-eaeb-40da-98e6-a408c7342f4c)
i tried set it to false and then the picture diappered, just the same as your issue
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/c34b22fd-5bf6-46b4-9427-ca1d55244fc2)
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/fa06d93e-d937-4417-9cbf-b78c6db5f88c)

**Comment by dsiguero** [comment]
That's it! Thank you so much @Lovirr!

It was relying on the default value (`false`). After setting it to `true` images now render inside quotes.

**Reply by Lovirr** [reply]
😄you are welcome！
